### ISSUE TITLE
NTR: Liver biopsy: iron increased in reticuloendothelial cells (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9154]
New term request Liver biopsy: iron increased in reticuloendothelial cells (elthat_100602220921): Macrophages; Perls' Prussian blue stain; hepatic biopsy  UMLS: 0282193

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000713
- label: Hepatic reticuloendothelial system cell iron deposition
- parents: [Abnormal hepatic iron concentration (HP:0040134)](https://hpo.jax.org/app/browse/term/HP:0040134)
- pmid: PMID:21274866
- def: Abnormal accumulation of iron in reticuloendothelial cells of the liver. This feature can be appreciated on liver biopsy by Perls iron stain for granular iron deposition, based on agreement that only discernible hemosiderin granules represent significant iron deposition.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9154
- synonyms:
